drift/lib/src/runtime/query_builder/components/order_by.dart

```diff
@@ -37,10 +44,18 @@ class OrderingTerm extends Component {
     return OrderingTerm(expression: expression, mode: OrderingMode.desc);
   }
 
+  /// Creates an ordering term  to get a number of random rows
+  /// using sqlite random function.
+  OrderingTerm.random()
+      : _expression = null,
+        mode = OrderingMode.random;
```

Comment:
Create factory method instead of custom constructor (other modes is factories too)